core/src/main/scala/org/apache/spark/ui/storage/StorageTab.scala

```diff
@@ -79,4 +87,19 @@ class StorageListener(storageStatusListener: StorageStatusListener) extends Spar
   override def onUnpersistRDD(unpersistRDD: SparkListenerUnpersistRDD) = synchronized {
     _rddInfoMap.remove(unpersistRDD.rddId)
   }
+
+  override def onBlockUpdate(blockUpdateEvent: SparkListenerBlockUpdate) = synchronized {
+    // only update broadcast for now, need to be modified if want to track other blocks
```

Comment:
this comment is slightly misleading -- as in other places, I would instead point out that RDD blocks are handled in `onStageCompleted`, but for now we are ignoring all other blocks.